AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -3320,6 +3329,44 @@ private void displayRenderLoopDialog(Card mCurrentCard, RenderProcessGoneDetail
         }
     }
 
+    // show developer contact if js api used in card is deprecated
+    private void showDeveloperContact() {
+        View parentLayout = findViewById(android.R.id.content);
+        String snackbarMsg;
+        if (developerContact.equals("")) {
```

Comment:
Needs to handle the user supplying null here

```suggestion
        if (TextUtils.isEmpty(developerContact)) {
```